packages/tangle/timemanager.go

```diff
@@ -152,16 +160,44 @@ func (t *TimeManager) updateTime(messageID MessageID) {
 		if t.lastConfirmedMessage.Time.After(message.IssuingTime()) {
 			return
 		}
-
 		t.lastConfirmedMessage = LastConfirmedMessage{
-			MessageID: messageID,
-			Time:      message.IssuingTime(),
+			MessageID:     messageID,
+			Time:          message.IssuingTime(),
+			ConfirmedTime: time.Now(),
 		}
-
 		t.updateSyncedState()
 	})
 }
 
+// RCTT return relative current tangle time.
+func (t *TimeManager) RCTT() time.Time {
+	newRCTT := t.calculateNewRCTT()
+	t.lastRCTTMutex.Lock()
+	defer t.lastRCTTMutex.Unlock()
+	if newRCTT.After(t.lastRCTT) {
```

Comment:
When confirmations stop for some time, then RCTT will go forward, but then when confirmations resume, RCTT will stop until confirmations catch up with the last RCTT value (or at most the difference between new CTT and the last RCTT value) - is this a desired property?

Reply:
@daria305 can you participate?

Reply:
RCTT should always be ahead of CTT. So we do not want to have this check here.

Reply:
Done